science/projects/soroush_heidary/01-main-content/Main-Content.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def find_target_element(page): 
</pre><p>I'm not sure how you handle it when the description text appears more than once on the page. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>